x/crosschain/keeper/finalized_inbounds.go

```diff
@@ -0,0 +1,19 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/crosschain/types"
+)
+
+func (k Keeper) AddFinalizedInbound(ctx sdk.Context, intxHash string, chainID int64, eventIndex uint64) {
```

Comment:
Can we add unit tests for these?

Reply:
a79b057e6f84ef9166ff5b826aebc3267ba0802f